internal/pkg/cli/task_run.go

```diff
@@ -82,6 +82,8 @@ type runTaskVars struct {
 
 	envVars map[string]string
 	command string
+
+	defaultEnv bool
```

Comment:
nit:
```suggestion
	useDefaultCluster bool
```

Reply:
I changed it to `useDefaultSubnets` because when the user specify `--default`, we grab the default subnets. 

If the user specify `--subnets`, we use the default cluster as well. So default cluster applies both to the case where we want to use the default subnets and to the case where we use a specified subnet.